### PR TITLE
Revert "APP-88: [C] Integrate gpio demos for different platforms into single demo"

### PR DESCRIPTION
**Pull request by vadimol**
Reverts kaaproject/sample-apps#301

### CODE REVIEW

**Comment by Slash32** [comment]
@vadimol could you please add some description why reverting?

**Comment by vadimol** [comment]
The initial PR breaks the build of the sandbox. After reverting everything works fine. It looks like that there were incorrect changes in assembly files.

I doubt that initial PR was tested before submitting.

File: gpiocontrol/assembly/src-android.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 <assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
-    <id>gpiocontrol-android</id>
+    <id>gpiocontrol-src-android</id>
```
Comment (RostakaGmfun) [inline gpiocontrol/assembly/src-android.xml:22]: It might be the line which failed the build. I will open PR to check this.

**Comment by Slash32** [comment]
@kaa-leeroy test this hard please

**Comment by vadimol** [comment]
The issue was solved in the scope of 
https://github.com/kaaproject/sample-apps/pull/310

**Pull request closed without merging**